product/consul.go

```diff
@@ -7,9 +7,9 @@ import (
 	"github.com/hashicorp/go-hclog"
 
 	"github.com/hashicorp/hcdiag/client"
-	"github.com/hashicorp/hcdiag/op"
-	s "github.com/hashicorp/hcdiag/op"
-	logs "github.com/hashicorp/hcdiag/op/log"
+	"github.com/hashicorp/hcdiag/runner"
+	s "github.com/hashicorp/hcdiag/runner"
```

Comment:
I know this was not part of the PR, but I see that we're importing `runner` directly as well as aliased to `s`. Could we get rid of the alias? It looks like it's only used on line 40.

Reply:
There were a few places seeker was aliased - missed this one in the refactor, thanks!